rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy

```diff
@@ -20,8 +20,8 @@ class LocalJobFriendlyPackerCommandFactory implements PackerCommandFactory {
 
   @Override
   List<String> buildPackerCommand(String baseCommand, Map<String, String> parameterMap, String absoluteTemplateFilePath) {
-    def packerCommand = ["packer", "build", "-color=false"]
-
+    def packerCommand = [baseCommand, "packer", "build", "-color=false"]
+    packerCommand.removeAll([""])
```

Comment:
Is the `removeAll([""])` needed here? If `CloudProviderBakeHandler.getBaseCommand()` already returned `""`, I would have expected this to work ok without removing the empty string item.